docs/reference/configuration/Platform.md

```diff
@@ -94,4 +116,9 @@ Name: platform.thread-stats-enabled
     Description: Set to 1 to enable thread stats. When enabled the function mbed_stats_thread_get_each returns non-zero data. See mbed_stats.h for more information
     Defined by: library:platform
     No value set
+Name: platform.use-mpu
```

Comment:
This is still a bit too cryptic (or the Mbed OS config system is). So, how do I REALLY disable it? I think I need to  modify the mbed_app.json, but how? (Noobie question, playing the devils advocate here).